lib/xnvme_libconf.c

```diff
@@ -28,37 +35,29 @@ xnvme_libconf_fpr(FILE *stream, enum xnvme_pr opts)
 	wrtn += fprintf(stream, "xnvme_libconf:");
 
 	wrtn += fprintf(stream, "\n");
-	for (int i = 0; xnvme_libconf[i]; ++i) {
-		fprintf(stream, "  - '%s'\n", xnvme_libconf[i]);
+	wrtn += fprintf(stream, "  entries:\n");
+	for (int i = 0; libconf->entries[i]; ++i) {
+		fprintf(stream, "  - '%s'\n", libconf->entries[i]);
 	}
 
 	// TODO: this is kinda hackish... a better means for backends to provide
 	// various introspective information should be provided
```

Comment:
Let's remove this TODO. We already have an issue: #227

Reply:
Right, ok. It makes a note an another issue with the library-config implementation, I have added that as an addendum to https://github.com/xnvme/xnvme/issues/227 and removed the TODO.